src/index.js

```diff
@@ -0,0 +1,77 @@
+/* import _, { add } from 'lodash'; */
+import './style.css';
+import { loadDataLs, saveDataLs } from './modules/localStorage.js';
+import Task from './modules/Tasks.js';
+import { ChangeTaskStatus, clearList } from './modules/Interactive.js';
+
+let tasks = loadDataLs();
+Task.LoadList(tasks);
+
+const addContainer = document.querySelector('.addContainer');
+
+/* Add new tasks */
+const Addtask = (e) => {
+  e.preventDefault();
+  const description = addContainer.elements[0].value;
+  const count = tasks.length + 1;
+  const task = new Task(description, count);
+  tasks.push(task);
+  Task.LoadList(tasks);
+  saveDataLs(tasks);
+  addContainer.reset();
+};
+
+addContainer.addEventListener('submit', Addtask);
+
+const section = document.querySelector('.list');
+
+/* Remove task */
+section.addEventListener('click', (event) => {
+  const { target } = event;
+  const NewId = target.id;
+  if (target.type === 'submit') {
+    Task.removeObWithId(tasks, NewId * 1);
+    Task.renewIndex(tasks);
+    Task.LoadList(tasks);
+    saveDataLs(tasks);
+  }
+});
+
+/* Edit tasks */
+section.addEventListener('focusout', (event) => {
+  const { target } = event;
+  const editTask = target.value;
+  const idEdit = target.id;
+  const change = Task.editTask(editTask, tasks, idEdit);
+  if (change !== 0) {
+    Task.LoadList(tasks);
+    saveDataLs(tasks);
+  }
+});
+
+/* Complete tasks */
+section.addEventListener('click', (event) => {
+  const { target } = event;
+  if (target.type === 'checkbox') {
+    const id = target.id.replace(/\D/g, '');
+    const textTask = section.querySelector(`[name=i${id}]`);
+    if (target.checked === true) {
```

Comment:
- [x] Good job applying this conditional here, however, I think it would be better to use `if (target.checked)` instead of `if (target.checked === true)` since it is the same and it helps you keep your code simple (KISS).